### PR TITLE
Kits/Site Identity: Fix - Cannot upload svg logo.

### PR DESCRIPTION
**Pull request by Nevoss**

### CODE REVIEW

File: core/kits/documents/tabs/settings-site-identity.php [hunk 1]
```diff
@@ -26,6 +27,9 @@ protected function register_tab_controls() {
 		$site_icon_id = get_option( 'site_icon' );
 		$site_icon_src = wp_get_attachment_image_src( $site_icon_id, 'full' );
 
+		// If cannot upload svg normally, it will add a custom inline option to force svg upload if requested. (in logo and favicon)
+		$can_upload_unfiltered_images = Files_Upload_Handler::is_enabled();
```
Comment (matipojo) [inline core/kits/documents/tabs/settings-site-identity.php:31]:
```suggestion
		$include_svg_inline_option = ! Files_Upload_Handler::is_enabled();
```

**Review by matipojo — CHANGES_REQUESTED** [review]

File: core/kits/documents/tabs/settings-site-identity.php [hunk 2]
```diff
@@ -68,6 +72,7 @@ protected function register_tab_controls() {
 			[
 				'label' => __( 'Site Logo', 'elementor' ),
 				'type' => Controls_Manager::MEDIA,
+				'include_svg_inline_option' => ! $include_svg_inline_option,
```
Comment (matipojo) [inline core/kits/documents/tabs/settings-site-identity.php:75]:
```suggestion
				'include_svg_inline_option' => $include_svg_inline_option,
```

File: core/kits/documents/tabs/settings-site-identity.php [hunk 3]
```diff
@@ -26,6 +27,9 @@ protected function register_tab_controls() {
 		$site_icon_id = get_option( 'site_icon' );
 		$site_icon_src = wp_get_attachment_image_src( $site_icon_id, 'full' );
 
+		// If cannot upload svg normally, it will add a custom inline option to force svg upload if requested. (in logo and favicon)
+		$include_svg_inline_option = Files_Upload_Handler::is_enabled();
```
Comment (matipojo) [inline core/kits/documents/tabs/settings-site-identity.php:31]:
```suggestion
		$include_svg_inline_option = ! Files_Upload_Handler::is_enabled();
```

File: core/kits/documents/tabs/settings-site-identity.php [hunk 4]
```diff
@@ -81,6 +86,7 @@ protected function register_tab_controls() {
 			[
 				'label' => __( 'Site Favicon', 'elementor' ),
 				'type' => Controls_Manager::MEDIA,
+				'include_svg_inline_option' => ! $include_svg_inline_option,
```
Comment (matipojo) [inline core/kits/documents/tabs/settings-site-identity.php:89]:
```suggestion
				'include_svg_inline_option' => $include_svg_inline_option,
```

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by rodolphebertozzo — DISMISSED** [review]

File: core/kits/documents/tabs/settings-site-identity.php [hunk 5]
```diff
@@ -68,6 +72,7 @@ protected function register_tab_controls() {
 			[
 				'label' => __( 'Site Logo', 'elementor' ),
 				'type' => Controls_Manager::MEDIA,
+				'include_svg_inline_option' => $should_include_svg_inline_option,
```
Comment (matipojo) [inline core/kits/documents/tabs/settings-site-identity.php:75]:
```suggestion
				'should_include_svg_inline_option' => $should_include_svg_inline_option,
```

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by matipojo — DISMISSED** [review]

File: assets/dev/js/editor/controls/media.js [hunk 6]
```diff
@@ -44,9 +44,11 @@ ControlMediaItemView = ControlMultipleBaseItemView.extend( {
 		this.ui.controlMedia.toggleClass( 'elementor-media-empty', ! url );
 	},
 
-	openFrame: function() {
-		if ( ! FilesUploadHandler.isUploadEnabled( this.getMediaType() ) ) {
-			FilesUploadHandler.getUnfilteredFilesNotEnabledDialog( () => this.openFrame() ).show();
+	openFrame: function( e ) {
+		const mediaType = e?.target?.dataset?.value || this.getMediaType();
```
Comment (kobizz) [inline assets/dev/js/editor/controls/media.js:48]: It's better to call the parameter something like `media-type` rather than `value`

**Review by kobizz — CHANGES_REQUESTED** [review]

**Review by matipojo — APPROVED** [review]

**Review by kobizz — APPROVED** [review]

**Review by KingYes — APPROVED** [review]

**Pull request merged**